### PR TITLE
Article Reading Pull Request

### PR DESCRIPTION
**Pull request by mkedmonds**
Photo included is a background image for the webpage.

### CODE REVIEW

File: 2021-09-27-retro-madelineedmonds.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            margin: auto auto;
+            background-color: white;
+            border: 5px solid black;
+            padding: 25px;
+        }
+
+        h1 {
+            font-size: 60px;
+            padding: 35px;
+            /* text-align:; */
```
Comment (ogennadi) [inline 2021-09-27-retro-madelineedmonds.html:25]: Please remove the commented out code since it does nothing

**Comment by ogennadi** [comment]
Please add your time logs to the HTML file

**Comment by mkedmonds** [comment]
test

**Comment by mkedmonds** [comment]
t

**Pull request closed without merging**